lib/data/replacements.json

```
@@ -670,6 +671,18 @@
       }
     ]
   },
+  "gradle-wrapper-validation-action": {
+    "description": "gradle/wrapper-validation-action got renamed to gradle/actions/wrapper-validation",
+    "packageRules": [
+      {
+        "matchCurrentVersion": "/^3(\\.)?/",
```

```suggestion
        "matchCurrentVersion": "/^3/",
```
would achieve the same in a more concise form. With `(\\.)?` a dot may or may not follow after `3`. The outcome is the same though.